java/clients/src/test/java/sleeper/status/report/compactionjob/StatusReporterUnfinishedQueryTest.java

```diff
@@ -34,129 +30,26 @@
 public class StatusReporterUnfinishedQueryTest extends StatusReporterTestBase {
 
     @Test
-    public void shouldReportCompactionJobStatusWithCreatedJobs() throws Exception {
+    public void shouldReportCompactionJobStatusForStandardAndSplittingCompactionsInRange() throws Exception {
```

Comment:
This should be renamed to `shouldReportCompactionJobStatusForStandardAndSplittingCompactionsUnfinished`

Reply:
Done